q2_diversity/tests/test_ordination.py

```diff
@@ -52,3 +53,37 @@ def test_pcoa_biplot(self):
 
         skbio.util.assert_ordination_results_equal(observed, expected,
                                                    ignore_directionality=True)
+
+
+class tsneTests(unittest.TestCase):
```

Comment:
A general note - we tend to use the "arrange - act - assert" pattern with respect to unit tests. You don't need to rewrite anything here, but I wanted to show you what this might look like - we like it, and think its a useful pattern:

```def test_tsne_default(self):
    observed = tsne(self.dm)

    self.assertEqual(observed.samples.shape, (3, 3))
    self.assertEqual(something else...)
```